DependencyInjection/Configuration.php

```diff
@@ -90,6 +90,20 @@ public function getConfigTreeBuilder()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('discovery')
+                    ->addDefaultsIfNotSet()
+                    ->info('Control what clients should be found by the discovery.')
+                    ->children()
+                        ->scalarNode('client')
+                            ->defaultValue('auto')
+                            ->info('Set to "auto" to see auto discovered client in the web profiler. If provided a service id for a client then this client will be found by auto discovery.')
+                        ->end()
+                        ->scalarNode('async_client')
+                            ->defaultNull()
```

Comment:
I added it in the docs: 
https://github.com/php-http/documentation/pull/118/files#diff-33b9b1161d0ce83931e73c48bf9e6900R71